src/main/java/org/broadinstitute/hellbender/tools/copynumber/PostprocessGermlineCNVCalls.java

```diff
@@ -171,6 +179,20 @@
     )
     private List<String> allosomalContigList;
 
+    @Argument(
+            doc = "Input VCF with combined intervals for all samples",
+            fullName = INPUT_INTERVALS_LONG_NAME,
+            optional = true
+    )
+    private File combinedIntervalsVCFFile = null;
```

Comment:
File inputs should be `GATKPath`s (also in `JointGermlineCNVSegmentation`)

Reply:
Since this is going to go to the underlying Python script as-is, I think it has to stay a File right?  I suspect it will crash and burn if it gets a GCS path.